From 0c87e9f4aacbefa318b55e71cde64d4dca4efb8b Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Tue, 5 Sep 2017 16:16:01 +0800
Subject: [PATCH 3306/5242] MLK-17491-1 serial: fsl_lpuart: fix the invalid
 comments

commit  9e65e2f6e123e20631f653771c55f242166976ea from
https://source.codeaurora.org/external/imx/linux-imx.git

This part of code is derived from Kinetis and is obviously invalid for ULP,
So delete it to avoid confusing.

Acked-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/tty/serial/fsl_lpuart.c |    8 --------
 1 file changed, 8 deletions(-)

diff --git a/drivers/tty/serial/fsl_lpuart.c b/drivers/tty/serial/fsl_lpuart.c
index 10947b3..c25b9f2 100644
--- a/drivers/tty/serial/fsl_lpuart.c
+++ b/drivers/tty/serial/fsl_lpuart.c
@@ -1820,14 +1820,6 @@ static void lpuart32_shutdown(struct uart_port *port)
 		}
 	}
 
-	/*TODO handle buadrate outside acceptable rate
-	 * if (baudDiff > ((config->baudRate_Bps / 100) * 3))
-	 * {
-	 *    Unacceptable baud rate difference of more than 3%
-	 *    return kStatus_LPUART_BaudrateNotSupport;
-	 * }
-	 *
-	 */
 	tmp = lpuart32_read(&sport->port, UARTBAUD);
 
 	if ((osr > 3) && (osr < 8))
-- 
1.7.9.5

